index.js

```diff
@@ -126,13 +126,9 @@ cm.Item({
 });
 
 function checkLocation(opts) {
```

Comment:
I'm not really seeing a use for this function, the only place we use it also checks that the url is a vine url. I don't know why we wouldn't want to launch min-vid from urls of that type when not on twitter or vine.com. Let's remove it and the call to is above.